Reaqtor/Samples/Shebang/Reaqtor.Shebang.Client/QueryOperators.Generated.cs

```diff
@@ -450,6 +468,24 @@ public static IAsyncReactiveQbservable<int> Count<TSource>(this IAsyncReactiveQb
             return source.Provider.CreateQbservable<int>(Expression.Call(((MethodInfo)MethodBase.GetCurrentMethod()).MakeGenericMethod(typeof(TSource)), source.Expression, predicate));
         }
 
+        [KnownResource("rx://observable/defaultifempty")]
+        public static IAsyncReactiveQbservable<TSource> DefaultIfEmpty<TSource>(this IAsyncReactiveQbservable<TSource> source)
+        {
+            if (source == null)
+                throw new ArgumentNullException(nameof(source));
+
+            return source.Provider.CreateQbservable<TSource>(Expression.Call(((MethodInfo)MethodBase.GetCurrentMethod()).MakeGenericMethod(typeof(TSource)), source.Expression));
+        }
+
+        [KnownResource("rx://observable/defaultifempty/defaultValue")]
+        public static IAsyncReactiveQbservable<TSource> DefaultIfEmpty<TSource>(this IAsyncReactiveQbservable<TSource> source, TSource defaultValue)
+        {
+            if (source == null)
+                throw new ArgumentNullException(nameof(source));
+
+            return source.Provider.CreateQbservable<TSource>(Expression.Call(((MethodInfo)MethodBase.GetCurrentMethod()).MakeGenericMethod(typeof(TSource)), source.Expression, Expression.Constant(defaultValue, typeof(TSource))));
```

Comment:
See my earlier remark. The code generated for Constant here is a good example of specifying the type.